tests/functional/Ticket.php

```diff
@@ -5555,4 +5555,58 @@ public function testGetEntitiesForRequesters()
         $entities = $ticket->getEntitiesForRequesters(["_users_id_requester" => $user_id]);
         $this->array($entities)->isIdenticalTo([$entity2_id, $entity1_id]);
     }
+
+    public function testShowCentralCountCriteria()
```

Comment:
A test should be added for `Ticket::showCentralCountCriteria(false)` (i.e. for counters displayed in global view).